src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -168,7 +168,7 @@ public void saveAddressBook(ReadOnlyAddressBook addressBook, Path filePath)
         // Triggers the saveAddressBook method by executing an add command
         String addCommand = AddCommand.COMMAND_WORD + NAME_DESC_AMY + PHONE_DESC_AMY
                 + EMAIL_DESC_AMY + ADDRESS_DESC_AMY + NOTE_DESC_AMY;
-        Person expectedPerson = new PersonBuilder(AMY).withTags().build();
+        Person expectedPerson = new PersonBuilder(AMY).withTags().removeLevel().build();
```

Comment:
What is removeLevel() here doing?

Reply:
PersonBuilder(AMY) now adds a level by default, so removeLevel is used here to match the expectedPerson with the person created by addCommand, which lacks a level.